modules/ui/sections/map_daterange.js

```diff
@@ -0,0 +1,167 @@
+import { dispatch as d3_dispatch } from 'd3-dispatch';
+import { select as d3_select } from 'd3-selection';
+
+import { t } from '../../core/localizer';
+import { uiTooltip } from '../tooltip';
+import { uiSection } from '../section';
+import { utilQsString, utilStringQs } from '../../util';
+
+const DEFAULT_MIN_DATE = '-4000-01-01';
+const DEFAULT_MAX_DATE = (new Date()).getFullYear() + '-12-31';
```

Comment:
I’d be in favor of relaxing this constraint at some point, so that users don’t get trapped in a situation where the feature they drew is no longer accessible in iD and it isn’t clear why: OpenHistoricalMap/issues#459. But this constraint was in the previous implementation, so I can bring this up in a separate issue.

Reply:
This min/max is not enforced by the Date Range boxes, is just the starting/default values. I should rename them `DEFAULT_START_DATE` and `DEFAULT_END_DATE`, since they're what's in the box and not an enforced min/max.

Reply:
Great, then that pretty much addresses the issue I reported in OpenHistoricalMap/issues#459.